src/Song.cpp

```diff
@@ -1768,26 +1768,22 @@ RString Song::GetCacheFile(RString sType)
 RString Song::GetFileHash()
 {
 	if (m_sFileHash.empty()) {
-		RString sPath = SetExtension(GetSongFilePath(), "sm");
-		if (!IsAFile(sPath))
-			sPath = SetExtension(GetSongFilePath(), "dwi");
-		if (!IsAFile(sPath))
-			sPath = SetExtension(GetSongFilePath(), "sma");
-		if (!IsAFile(sPath))
-			sPath = SetExtension(GetSongFilePath(), "bms");
-		if (!IsAFile(sPath))
-			sPath = SetExtension(GetSongFilePath(), "ksf");
-		if (!IsAFile(sPath))
-			sPath = SetExtension(GetSongFilePath(), "json");
-		if (!IsAFile(sPath))
-			sPath = SetExtension(GetSongFilePath(), "jso");
-		if (!IsAFile(sPath))
-			sPath = SetExtension(GetSongFilePath(), "ssc");
-		if (IsAFile(sPath))
+		return m_sFileHash;

```

Comment:
Um this seems to now do the opposite of what we want:

Previous was:

if hash empty:
  compute spath
  compute hash
  
Now we're just returning the an empty hash if it's empty.